.github/workflows/cffconvert.yml

```
@@ -0,0 +1,19 @@
+name: cffconvert
+
+on:
+  push:
+    paths:
+      - CITATION.cff
```

Let's also run this on pull requests:
```suggestion
  pull_request:
    paths:
      - CITATION.cff
  push:
    paths:
      - CITATION.cff
```

---

Done, thanks for the suggestion.